public/index.html

```diff
@@ -1,189 +1,216 @@
+              <input type="checkbox" id="transform" class="form-check-input" checked>
+              <label for="transform" class="form-check-label">Simplify diagram</label>
+            </div>
+            <div class="form-group form-check" data-bs-toggle="tooltip"
+              title="Enables checking for common problems using the Verilator compiler.">
+              <input type="checkbox" id="lint" class="form-check-input" checked>
+              <label for="lint" class="form-check-label">Lint source code using <a
+                  href="https://verilator.org/">Verilator</a></label>
+            </div>
+            <div class="form-group" data-bs-toggle="tooltip"
+              title="Changes how the circuit elements are automatically positioned after synthesis.">
+              <label for="layout">Layout engine</label>
+              <select id="layout" class="form-control">
+                <option value="elkjs">ElkJS (more readable)</option>
+                <option value="dagre">Dagre (legacy)</option>
+              </select>
+            </div>
+            <div class="form-group" data-bs-toggle="tooltip"
+              title="Changes how the synthesized circuit is simulated. The synchronous engine is well tested, but it's also very slow.">
+              <label for="engine">Simulation engine</label>
+              <select id="engine" class="form-control">
+                <option value="worker">WebWorker (faster and responsive)</option>
+                <option value="synch">Synchronous (extensible but slow)</option>
+              </select>
+            </div>
+            <div class="form-group" data-bs-toggle="tooltip"
+              title="Enables finite state machine processing in Yosys. This corresponds to the 'fsm' and 'fsm -nomap' Yosys commands.">
+              <label for="fsm">FSM transform (experimental)</label>
+              <select id="fsm" class="form-control">
+                <option value="">No FSM transform</option>
+                <option value="yes">FSM transform</option>
+                <option value="nomap">FSM as circuit element</option>
+              </select>
+            </div>
+            <div class="form-group form-check" data-bs-toggle="tooltip"
+              title="This corresponds to the 'fsm_expand' Yosys command.">
+              <input type="checkbox" id="fsmexpand" class="form-check-input">
+              <label for="fsmexpand" class="form-check-label">Merge more logic into FSM</label>
+            </div>
+          </form>
+        </div>
+        <div role="tabpanel" class="tab-pane tab-padded" id="iopanel">
+        </div>
+      </div>
+      <div id="synthesize-bar">
+        <form>
+          <button type="submit" class="btn btn-primary">Synthesize and simulate!</button>
+        </form>
+      </div>
+    </div>
+    <div id="gutter_horiz" class="gutter gutter-horizontal"></div>
+    <div id="paper">
+    </div>
+    <div class="zoom-buttons-wrapper">
```

Comment:
The zoom buttons are only present on the main paper. It would be better if they were available also on subcircuit popups. (Same thing applies to the SVG saving feature.)